content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,253 @@
+   :guilabel:`Control Policy` field.
 
-From the **Apps** application, search for the **Purchase** module and
-install it. Due to certain dependencies, installing Purchase will
-automatically install the **Inventory** and **Accounting** applications.
+.. image:: manage/manage-product-form.png
+   :align: center
+   :alt: Control policy field on product form.
 
-Creating products
------------------
+3-way matching
+--------------
 
-Creating products in Odoo is essential for quick and efficient
-purchasing within Odoo. Simply navigate to the Products submenu under
-Purchase and click create.
+:guilabel:`3-way matching` ensures vendor bills are only paid once some (or all) of the products
+included in the purchase order have actually been received.
 
-.. image:: manage/manage01.png
-  :align: center
+To activate it, go to :menuselection:`Purchase --> Configuration --> Settings`, and scroll down to
+the :guilabel:`Invoicing` section. Then, check the box next to
+:guilabel:`3-way matching: purchases, receptions, and bills`, and click :guilabel:`Save`.
 
-When creating the product, pay attention to the **Product Type** field,
-since it is important:
+.. important::
+   :guilabel:`3-way matching` is *only* intended to work with the :guilabel:`Bill Control` policy
+   set to :guilabel:`Received quantities`.
 
-- Products that are set as **Stockable or Consumable** will allow you
-  to keep track of their inventory levels. These options imply
-  stock management and will allow for receiving these kinds of
-  products.
+.. image:: manage/manage-three-way-matching.png
+   :align: center
+   :alt: Activated three-way matching feature in purchase settings.
 
-- Conversely, products that are set as a **Service or Digital Product**
-  will not imply stock management, simply due to the fact that
-  there is no inventory to manage. You will not be able to receive
-  products under either of these designations.
+Create (and manage) vendor bills on receipts
+============================================
 
-.. tip::
-   It is recommended that you create a **Miscellaneous** product for all purchases
-   that occur infrequently and do not require inventory valuation or management.
-   If you create such a product, it is recommended to set the product type to **Service**.
+When products are received into a company's warehouse, receipts are created. Once the company
+processes the received quantities, they can choose to create a vendor bill directly from the
+wareouse receipt form. Depending on the *bill control* policy chosen in the settings, vendor bill
+creation is completed at different steps of the procurement process.
 
-Managing your Vendor Bills
-==========================
+With the bill control policy set to ordered quantities
+------------------------------------------------------
 
-Purchasing products or services
--------------------------------
+To create and manage vendor bills on receipts using the *ordered quantities* bill control policy,
```

Comment:
Don't think we use a gui here since we're not there on the screen. We're emphasizing the field/setting that we will arrive to after the dashboard instruction.